src/exchange/disclaimer/disclaimer.controller.js

```diff
@@ -28,44 +25,73 @@ angular
         }
         /* eslint-enable class-methods-use-this */
 
-        /* eslint-disable class-methods-use-this */
-        hasFullSlot (list) {
-            return _.some(list, (item) => !item.emptySlotFlag);
+        _getDisclaimers ({ pageSize, offset }) {
+            return this.services
+                .Exchange.getDisclaimers(this.organization, this.productId, pageSize, offset - 1);
         }
-        /* eslint-enable class-methods-use-this */
-
-        loadPaginated (count, offset) {
-            this.loading.table = true;
 
-            this.services
-                .Exchange.getDisclaimers(this.$routerParams.organization, this.$routerParams.productId, count, offset)
+        refreshList () {
+            if (!this.disclaimersList) { return undefined; }
```

Comment:
Sometimes you return `undefined` and sometimes you return a Promise, maybe you can make sure that the function always returns a Promise :smiley:

Reply:
This is because of our Eslint config, I would rather not return anything, all the time. I return undefined, when I don't want my function to execute.

Reply:
It's alright as long as nobody's using the promise returned by the function.  In this case everything looks good